src/pois.rs

```diff
@@ -54,7 +54,9 @@ pub struct IndexedPoi {
 impl IndexedPoi {
     pub fn from_row(row: Row, langs: &[String]) -> Option<IndexedPoi> {
         let id: String = row.get("id");
-        let name: String = row.get("name");
+        let name = row
+            .get::<_, Option<String>>("name")
+            .unwrap_or_else(|| "".to_string());
```

Comment:
`unwrap_or_default()` may be slightly more readable in this case ?

Reply:
`unwrap_or_default` looks cleaner indeed, let's go for it.